tests/unit/utils/test_json_tools.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from utils.json_tools import parse_json_markdown
+
+
+correct_inputs = (
+    ("{}", {}),
+    ('{"x":42}', {"x": 42}),
+    ('{"x":42, "y":""}', {"x": 42, "y": ""}),
+    ('   {"x":42, "y":""}   ', {"x": 42, "y": ""}),
+    ('```{"x":42, "y":""}```', {"x": 42, "y": ""}),
+    ('\n{"x":42, "y":""}\n', {"x": 42, "y": ""}),
+    ('\t{"x":42, "y":""}\t', {"x": 42, "y": ""}),
+    ('```json{"x":42, "y":""}```', {"x": 42, "y": ""}),
+)
+
+
+@pytest.mark.parametrize("json,expected", correct_inputs)
+def test_parse_json_markdown(json, expected):
+    """Check parsing correct input."""
+    assert parse_json_markdown(json) == expected
+
+
+incorrect_inputs = (None, "", "foo", "\n", '"')
```

Comment:
how about a test in which the content inside the markdown/backticks is not json?

Reply:
yup, added